types/msg.go

```diff
@@ -6,49 +6,15 @@ import (
 
 //------- Results / Msgs -------------
 
-// HandleResult is the raw response from the handle call.
-// This is mirrors Rust's ContractResult<HandleResponse>.
-type HandleResult struct {
-	Ok  *HandleResponse `json:"ok,omitempty"`
-	Err string          `json:"error,omitempty"`
+// Result is the raw response from the init/handle/migrate calls.
+// This is mirrors Rust's ContractResult<Response>.
+type ContractResult struct {
```

Comment:
Nice name. Just make sure you update the name in the godocs to be the same.

(Go linter complains if the docstring doesn't start with the exact name of the type)

Reply:
Well spotted. I used Result first but then decided to use something less generic.